rosetta-ide/rosetta.tmLanguage.yaml

```diff
@@ -88,6 +89,16 @@ repository:
         - include: '#comment'
         - include: '#string'
 
+      override:
+        name: meta.override.rosetta
+        begin: '{{wordStart}}override{{wordEnd}}'
+        beginCaptures:
+          0: { name: keyword.other.override.rosetta }
+        end: '{{rootEnd}}'
+        patterns:
+        - include: '#comment'
+        - include: '#string'
+
```

Comment:
Could be better integrated inside `import`, but how about an already slight improvement?
```
      override:
        name: keyword.other.override.rosetta
        match: '{{wordStart}}override{{wordEnd}}'
```